src/main/java/app/models/passport/passportRFForeign/PassportRFForeign.java

```diff
@@ -0,0 +1,16 @@
+package app.models.passport.passportRFForeign;
+
+import app.models.passport.BasePassport;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+public class PassportRFForeign extends BasePassport {
+    @NotNull
+    private Date expirationDate;
+    @Digits(integer = 8, fraction = 0, message = "Invalid Passport number")
```

Comment:
This is a controller level validation, but the class is in `model` package.
Consider separating models and controller's DTOs (requests and responses)